src/test/java/com/datadoghq/sketch/ddsketch/footprint/Distributions.java

```diff
@@ -0,0 +1,59 @@
+package com.datadoghq.sketch.ddsketch.footprint;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
+
+public enum Distributions {
+    POINT {
```

Comment:
Nit.: `DIRAC_DELTA` may be more appropriate.

Reply:
I did actually consider using that name but thought `POINT` was more accessible - it's unlikely that someone understands what the Dirac delta function is unless they've taken some university level maths.